components/accelerator/nvidia/gpm/component.go

```diff
@@ -78,14 +91,53 @@ func (c *component) Metrics(ctx context.Context, since time.Time) ([]components.
 	}
 
 	ms := make([]components.Metric, 0, len(smOccupancies))
-	for _, m := range smOccupancies {
-		ms = append(ms, components.Metric{
-			Metric: m,
-			ExtraInfo: map[string]string{
-				"gpu_id": m.MetricSecondaryName,
-			},
-		})
+	ms = updateMetrics(ms, smOccupancies)
+	intUtils, err := nvidia_query_metrics_gpm.ReadGPUIntUtilPercents(ctx, since)
+	if err != nil {
+		return nil, fmt.Errorf("fail to read sm int util: %w", err)
+	}
+	ms = updateMetrics(ms, intUtils)
+	anyTensorUtils, err := nvidia_query_metrics_gpm.ReadGPUAnyTensorUtilPercents(ctx, since)
```

Comment:
```suggestion
	ms = updateMetrics(ms, smOccupancies)
	intUtils, err := nvidia_query_metrics_gpm.ReadGPUIntUtilPercents(ctx, since)
	if err != nil {
		return nil, fmt.Errorf("fail to read sm int util: %w", err)
	}
	ms = updateMetrics(ms, intUtils)

	anyTensorUtils, err := nvidia_query_metrics_gpm.ReadGPUAnyTensorUtilPercents(ctx, since)
```